docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -97,40 +146,166 @@ The sequela associated with the dietary iron deficiency cause in GBD 2017
-	* - Males and Females 1 month - 5 years
-	  - 100-109
-	  - 70-99
-	  - <70
-	* - Males and Females 5-14 years
-	  - 110-114
-	  - 80-109
-	  - <80
-	* - Males 15+ years
-	  - 110-129
-	  - 80-109
-	  - <80
-	* - Females 15+ years, non-pregnant 
-	  - 110-119
-	  - 80-109
-	  - <80
-	* - Females 15+ years, pregnant
-	  - 100-109
-	  - 70-00
-	  - <70
+The iron deficiency risk factor in GBD 2017 is a **continuous measure** modeled 
+as a distribution of hemoglobin concentrations *among the population with 
+conditions that cause iron deficiency anemia.* Below is a list of such causes, along with the list of sequela IDs that include anemia health states:
+
+.. list-table:: Causes 
+	:widths: 40 40 40
+	:header-rows: 1
+
+	* - Cause
+	  - Cause ID
+	  - Anemia-Afflicated Sequela ID
+	* - Dietary Iron Deficiency
+	  - 390
+	  - 206, 207, 208
+	* - Endocrine, Metabolic, Blood, and Immune Disorders
+	  - 619
+	  - 537, 538, 539
+	* - Uterine Fibroids
+	  - 604
+	  - 1106, 1107, 1108
+	* - Other Gynecological Diseases
+	  - 612
+	  - 525, 526, 527
+	* - Hookworm disease
+	  - 363
+	  - 172, 173, 174
+	* - Schistosomiasis
+	  - 351
+	  - 144, 145, 146
+	* - Other Neglected Tropical Diseases
+	  - 365
+	  - 177, 178, 179
+	* - Other Unspecified Infectious Diseases
+	  - 961
+	  - 240, 241, 242
+	* - Maternal Hemorrhage
+	  - 367
+	  - 182, 183, 184
+	* - Vitamin A Deficiency
+	  - 389
+	  - 5393, 5396, 5399
+	* - Peptic Ulcer Disease
+	  - 527
+	  - 4952, 4955, 4958, 4961, 4964, 4967, 4976, 4979, 4982, 5627, 5630, 5633, 7202, 7205, 7208
+	* - Gastritis and Duodenitis
+	  - 528
+	  - 4985, 4988, 4991, 4994, 4997, 5000, 5009, 5012, 5015, 5678, 5681, 5684, 7214, 7217, 7220
+	* - Chronic Kidney Disease
+	  - 589 (591, 592, 593, 997, 998)
+	  - 1004, 1005, 1006, 1008, 1009, 1010, 1012, 1013, 1014, 1016, 1017, 1018, 1020, 1021, 1022, 1024, 1025, 1026, 1028, 1029, 1030, 1032, 1033, 1034, 1361, 1364, 1367, 1373, 1376, 1379, 1385, 1388, 1391, 1397, 1400, 1403, 1409, 1412, 1415, 1421, 1424, 1427, 1433, 1436, 1439, 1445, 1448, 1451, 5213, 5216, 5219, 5222, 5225, 5228, 5237, 5240, 5243, 5246, 5249, 5252, 5261, 5264, 5267, 5270, 5273, 5276
+
+.. note::
+
+	According to the GBD modelers, ESRD - Dialysis, Crohn's disease, and 
+	ulcerative colitis were also included in this list, although there do not 
+	appear to be results for these causes in GBD 2017. Additionally, according 
+	to the GBD modelers, cirrhosis should be included in this list, although 
+	there do not appear to be any anemia-afflicted sequelae with results in GBD 
+	2017 within any of the cirrhosis causes.
+
+The distribution of hemoglobin concentrations represented by the iron 
+deficiency risk factor is assumed to have the same shape as the distribution of 
+hemoglobin concentrations in the entire population (anemia envelope), i.e. 40% 
+gamma and 60% mirror Gumbel ensemble distribution. The mean value for the iron 
+deficiency risk factor is stored under modelable entity ID 10487 (also REI ID 
```

Comment:
Interesting. I thought this was the mean for the total population, but I see in your code you checked that these were (almost) equal. Now I'm even more confused about what they did...

Reply:
Hmm, it looks like you are right - MEID 10487 is actually the mean in the total population according to the anemia modeler's source code. I thought that it was the risk factor population. Now i am confused too. I will look into this a little more and see what I can figure out. Darn!

Reply:
Ok, so based on what I heard back from Will, the information in this PR is no longer accurate. I will close it and upload an accurate version. Basically, I think what we are going to have to do is what James originally suggested, which is the approach of evaluating the probability that an individual is iron responsive given that they have severe anemia. We can use the iron-responsive cause list that Jordan provided to look into these conditional probabilities. However, I will also need to investigate why the ensemble distribution still is not matching the anemia impairment distribution in the figures that I have been sharing on slack.

Reply:
@alibow I'd say, instead of closing this PR, why not go ahead and merge it, but add a Warning or a Todo to note where the information is inaccurate and needs updating? I think it would be useful to get this information into the documentation, even if it's not perfect yet.

Reply:
Unfortunately while this is what I intended to do, I made a git process issue and accidentally committed my new changes to this branch that was still open. I could try to figure out how to revert back to the approved version, merge it, and then submit for a new PR, or we could just review the new commits on this PR